modules/installation-supported-aws-machine-types.adoc

```diff
@@ -29,6 +29,11 @@ endif::[]
 
 The following Amazon Web Services (AWS) instance types are supported with {product-title}.
 
+[NOTE]
+====
+It is suggested to use the machine types included in the following charts for your instances. If you use an instance type that is not listed in the chart, you need to ensure that the instance size you use matches the standard performance of the listed instance type. 
```

Comment:
I think 'it is suggested to' is unnecessary here, and removing them won't really change the meaning of the sentence. It'll make the sentence concise.